src/Particle/DynamicCoordinates.h

```diff
@@ -55,6 +56,14 @@ class DynamicCoordinates
 
   virtual void setAllParticlePos(const ParticlePos_t& R)         = 0;
   virtual void setOneParticlePos(const PosType& pos, size_t iat) = 0;
+  virtual void mw_copyActiveOldParticlePos(const RefVector<DynamicCoordinates>& coords_list,
```

Comment:
Definitions are needed here, and broadly for all the functions in DynamicCoordinates